logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCaseTest.kt

```diff
@@ -34,16 +34,41 @@ import io.mockative.mock
 import io.mockative.once
 import io.mockative.verify
 import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.asFlow
 import kotlinx.coroutines.flow.flowOf
 import kotlinx.coroutines.test.runTest
-import kotlin.test.Ignore
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 @OptIn(ExperimentalCoroutinesApi::class)
 class GetNotificationsUseCaseTest {
 
-    @Ignore
+    @Test
+    fun givenSyncStateIsPending_thenOnlyEphemeralNotificationsObserved() = runTest {
+        val (arrange, getNotifications) = Arrangement()
+            .withEphemeralNotification()
+            .withIncrementalSyncState(IncrementalSyncStatus.FetchingPendingEvents)
+            .withConnectionList(listOf())
+            .withConversationsForNotifications(listOf())
+            .arrange()
+
+        getNotifications().test {
+            awaitComplete()
```

Comment:
Will it ever finish?

`GetNotificationsUseCase` returns an endless flow that never completes.
I think we should await the first result is emitted, and then cancel the collection (to cancel the flow).